From: Michael Tretter <m.tretter@pengutronix.de>
Date: Thu, 10 Feb 2022 10:10:57 +0100
Subject: [PATCH] HACK: video: ssd1307fb: increase VCOMH to 0x40

Setting VCOMH to 0x40 is necessary to avoid an unbearable delay when
turning on the display.

Only 0x20 and 0x30 are documented as VCOMH, but Linux uses 0x40 and
documents that it is a undocumented value.

Signed-off-by: Michael Tretter <m.tretter@pengutronix.de>
---
 drivers/video/ssd1307fb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/ssd1307fb.c b/drivers/video/ssd1307fb.c
index 9e141a851f51..bae6daede58e 100644
--- a/drivers/video/ssd1307fb.c
+++ b/drivers/video/ssd1307fb.c
@@ -388,7 +388,7 @@ static struct ssd1307fb_deviceinfo ssd1307fb_ssd1305_deviceinfo = {
 };
 
 static struct ssd1307fb_deviceinfo ssd1307fb_ssd1306_deviceinfo = {
-	.default_vcomh = 0x20,
+	.default_vcomh = 0x40,
 	.default_dclk_div = 1,
 	.default_dclk_frq = 8,
 	.need_chargepump = 1,
